src/renderer.tsx

```diff
@@ -76,8 +76,6 @@ ipcRenderer.on(
     }: { isPackaged: string; locatePastelConfDir: string },
   ) => {
     if (isPackaged) {
-      console.log = log.log
-      console.error = log.error
```

Comment:
Why should we use log instead of console.log? 
Because it will use when you want to write something to the log file

is it causing any bug?
Yes, because if we writing all `console` to the log file, it may have sensitive info and we can't control/handle this case. Because we haven't managed the console.log/console.error in the development before which means if someone has used `console` in the code, it will write that data/info to the log file.

In the previous PR I suggested to use console.log instead of log, please advice if I'm missing something important
Yeah it a good suggestion and I updated base on your suggestion, But when I released a new version, I realized if we do like that we can't control the log file and the purpose of the log file is to write useful information, but it must be controlled. It means we need to know what we writing into the log file. So if someone wants to write something to the log file they must know exactly what they want to log.